website/pages/api-docs/features/filtering.mdx

```diff
@@ -9,83 +9,48 @@ description: |-
 
 # Filtering
 
-A filter expression is used to refine a data query for some API listing endpoints as notated in the individual API documentation.
-Filtering will be executed on the Consul server before data is returned, reducing the network load. To pass a
-filter expression to Consul, with a data query, use the `filter` parameter.
+Filter expressions refine data queries for some API listing endpoints, as notated in the individual API documentation.
+
+To create a filter expression, you will write one or more expressions. Each expression has a matching operators composed of selectors and values.
 
 ```shell
-curl -G <path> --data-urlencode 'filter=<filter expression>'
+curl -G <path> --data-urlencode 'filter=<"filter expression">'
```

Comment:
```suggestion
curl --get <path> --data-urlencode 'filter="<filter expression>"'
```
I feel the long form config option `--get` is a bit clearer than `-G`.

I realize this was already written like this prior to your change. I figure it might be worth changing while you're proposing these other modifications.